src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -176,15 +176,37 @@ class ArnoldOutput : public IECore::RefCounted
 				AiNodeSetStr( m_driver.get(), AiParamGetName( fileNameParameter ), output->getName().c_str() );
 			}
 
+			std::vector<std::string> headerVector;
 			for( IECore::CompoundDataMap::const_iterator it = output->parameters().begin(), eIt = output->parameters().end(); it != eIt; ++it )
 			{
 				if( boost::starts_with( it->first.string(), "filter" ) )
 				{
 					continue;
 				}
+
+				if( boost::starts_with( it->first.string(), "header:" ) )
+				{
+					const IECore::StringData *headerEntry = IECore::runTimeCast<const IECore::StringData>( it->second.get() );
+					if( headerEntry )
+					{
+						headerVector.push_back( "string " + it->first.string().substr(7, it->first.value().size()-7) + " " + headerEntry->readable().c_str() );
```

Comment:
We can omit the second argument to `substr()`, since the default is to give us to the end of the string, which is what we want.